CMakeLists.txt

```diff
@@ -17,8 +17,10 @@ LIST( APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake )
 INCLUDE_DIRECTORIES ( 
 	${CMAKE_CURRENT_SOURCE_DIR}/include
 	${CMAKE_CURRENT_BINARY_DIR}/include
+	${CMAKE_SOURCE_DIR}/tests
```

Comment:
Given the following diff, i kind of doubt that is actually the Problem. The only difference between the CMakeList of rbdl-orb and my fork of rbdl is the version.

```shell
$ diff rbdl/CMakeLists.txt rbdl-orb/CMakeLists.txt 
4,5c4,5
< SET ( RBDL_VERSION_MINOR 2 )
< SET ( RBDL_VERSION_PATCH 0 )
---
> SET ( RBDL_VERSION_MINOR 1 )
> SET ( RBDL_VERSION_PATCH 3 )
```

Reply:
Not sure what you are reffering to. In any version posterior to the introductino of these two lines, I cannot compile using the casadi backend, but it work just fine without them. I did not try to compile any other MODULE of RBDL though. I suspect there is a reason you introduced these two includes. That said, you introduced them using INCLUDE_DIRECTORIES instead of TARGET_INCLUDE_DIRECTORIES(target PRIVATE...) which is a valid solution too (I think)

Reply:
These two lines are present in the current version of rbdl-orb. Which as far as i am aware works fine for rbdl-casadi. It also does not make any sense to me that this could break compilation since it only specifies where to search for includes. Also the main CMakeList does not handle building of the casadi version of the library. The CMakeLists for that is located in the `casadi` subdirectory.

Reply:
I suspect the reason it does so is that in these folder there problably are includes for non casadi rbdl. I can give another shot, but removing these two lines made the linker happy here... If it was already there, I assume it is some kind of weird conflict... but in any case, you don't want to use INCLUDE_DIRECTORIES but you should use TARGET_INCLUDE_DIRECTORIES